### DISCUSSION TITLE
Soft deletes and relationships

### DISCUSSION THREAD

**Discussion by rikiless** [filamentphp/filament#5889] [Help] [answered]
I play with [Filament demo](https://github.com/filamentphp/demo) locally. I noticed that when I soft delete **brand** item it will update **products** used that brand (`shop.products.shop_brand_id`) to `null`. How is this achieved? I looked for policies/eloquent events and couldn't find anything.

**Comment by VincvanAs** [comment, marked as answer]
Check your migration file > nullOnDelete();?

Ref: https://laravel.com/docs/10.x/migrations#foreign-key-constraints

**Reply by rikiless** [reply]
Thanks. I didn't notice that filament demo is using SoftDeletes only in a few models, not everywhere.
